app/views/books/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for @book do |form| %>
```

Comment:
The `@book` here should be `book` so it uses the local variables passed into the partial